scripts/lib/spreadsheet-mappings.ts

```diff
@@ -132,6 +132,7 @@ export const VALUE_MAPPINGS: { [index: string]: AliasMap } = {
     ton: ['Ton'],
     watt: ['Watt'],
     btuh10k: ['Btuh10k'],
+    cmf50: ['CMF50'],
```

Comment:
I added this because PA-34 has the incentive "$200 rebate for air sealing. Air infiltration reduction (@ CFM50) x $0.25 up to a max of $200" ([source](https://www.pplelectricsavings.com/ppl/sites/ppl/files/2023-08/2023_PPL_Res_ResReb_3862941_FLY_Q3_PRJ101498.pdf)). I looked this up and I think it's a unit?